### ISSUE TITLE
Expand Support for Locations

### ISSUE THREAD

**Issue by dshanske** [TomerFi/home-assistant-custom-components#1]
https://www.hebcal.com/home/197/shabbat-times-rest-api

You are using what is referred to as the "Legacy Geoname Identifiers"

This doesn't cover all possible situations, and in fact is rather limiting. You may be better off supporting the geoname, zip, and pos which would get you everything. For position, you could pull the latitude and longitude of defined zones as well.

**Comment by TomerFi** [comment]
Yeah I thought about integrating geoid instead of the legacy geonames.
I'll try to make the time for it this weekend.
Thank you.
:-)
